lib/chef/provider/aws_s3_bucket.rb

```diff
@@ -74,13 +72,27 @@ def destroy_aws_object(bucket)
 
   private
 
+  def website_exist?(new_resource,bucket)
+    return true if new_resource.driver.s3.get_bucket_website(bucket: bucket.name)
+  rescue
```

Comment:
in principle, blind rescues should be avoided, especially without any logging. Is there a specific error we can rescue here?

Also, we're only returning false if we rescue and returning nil if an exception isn't raised. Are we sure that's never going to happen?